ostelco-ios-client/ViewControllers/Home/HomeViewController.swift

```diff
@@ -56,7 +56,7 @@ class HomeViewController: ApplePayViewController {
 
     private func checkForSubscription(_ products: [Product]) -> Bool {
         // See if the list contains offers.
-        let hasOffers = products.contains { $0.type == "offer" }
+        let hasOffers = products.contains { $0.type != "plan" && $0.currency.lowercased() == "sgd" }
```

Comment:
Might be worth throwing in a `TODO` to get rid of the singapore dollars check when we take this international